latest master (20210505) - ionic QS broken / cli version not found.

https://github.com/opendevstack/ods-quickstarters/pull/577 seems to have caused 

```
[0;32m    amazon-ebs: [Pipeline] sh[0m
[0;32m    amazon-ebs: + npm install -g ionic@6.13.1[0m
[0;32m    amazon-ebs: npm ERR! code ETARGET[0m
[0;32m    amazon-ebs: npm ERR! notarget No matching version found for ionic@6.13.1.[0m
[0;32m    amazon-ebs: npm ERR! notarget In most cases you or one of your dependencies are requesting[0m
[0;32m    amazon-ebs: npm ERR! notarget a package version that doesn't exist.[0m
[0;32m    amazon-ebs:[0m
[0;32m    amazon-ebs: npm ERR! A complete log of this run can be found in:[0m
```

@felipecruz91  - seems like the cli upgrade (did you test that?) fails the buidl now .. please take a  look asap

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,7 +1,7 @@
 # Changelog
 
 ### Modified
-- Added correct ionic package in Jenkinsfile
+- Added correct ionic package in Jenkinsfile ([#580](https://github.com/opendevstack/ods-quickstarters/pull/581))
 
 ### Modified
 - Removed forcing eslint configuration as it is default ([#573](https://github.com/opendevstack/ods-quickstarters/pull/578))

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,9 @@
 # Changelog
 
 ### Modified
+- Added correct ionic package in Jenkinsfile
+
+### Modified
 - Removed forcing eslint configuration as it is default ([#573](https://github.com/opendevstack/ods-quickstarters/pull/578))
 ### Modified
 - Default linter for Ionic is now eslint as tslint is deprecated ([#573](https://github.com/opendevstack/ods-quickstarters/pull/575))

--- a/fe-ionic/Jenkinsfile
+++ b/fe-ionic/Jenkinsfile
@@ -22,7 +22,7 @@ odsQuickstarterPipeline(
 ) { context ->
 
   stage("update ionic cli") {
-    sh "npm install -g ionic@${ionicVersion} && ionic version"
+    sh "npm install -g @ionic/cli@${ionicVersion} && ionic version"
     sh "npm install -g eslint@${eslintVersion} && eslint -v"
   }
 

